src/ObjCRuntime/TrampolineBlockBase.cs

```diff
@@ -7,26 +7,29 @@
 namespace ObjCRuntime {
 
 	public abstract class TrampolineBlockBase {
-			protected IntPtr blockPtr;
-
-			[DllImport ("/usr/lib/libobjc.dylib")]
-			static extern IntPtr _Block_copy (IntPtr ptr);
-
-			protected unsafe TrampolineBlockBase (BlockLiteral *block)
-			{
-				blockPtr = _Block_copy ((IntPtr) block);
-			}
-			
-			~TrampolineBlockBase ()
-			{
-				Runtime.ReleaseBlockOnMainThread (blockPtr);
-			}
-
-			protected unsafe static object GetExistingManagedDelegate (IntPtr block)
-			{
-				if (!BlockLiteral.IsManagedBlock (block))
-					return null;
-				return ((BlockLiteral *) block)->Target;
-			}
+
+		readonly IntPtr blockPtr;
+
+		[DllImport ("/usr/lib/libobjc.dylib")]
+		static extern IntPtr _Block_copy (IntPtr ptr);
+
+		protected unsafe TrampolineBlockBase (BlockLiteral *block)
+		{
+			blockPtr = _Block_copy ((IntPtr) block);
+		}
+
+		~TrampolineBlockBase ()
+		{
+			Runtime.ReleaseBlockOnMainThread (blockPtr);
+		}
+
+		protected IntPtr BlockPointer { get; }
```

Comment:
This is never set.

Reply:
yup, that was fixed in a later commit - not sure why the previous one was not cancelled (it usually is when a new commit is made to the branch)